pkg/cmd/repo/list/list.go

```diff
@@ -154,6 +154,10 @@ func listRun(opts *ListOptions) error {
 		return err
 	}
 
+	if listResult.Owner == "" {
+		return fmt.Errorf("owner / repo not found")
```

Comment:
Hi, I feel like the message `owner / repo not found` is not the best because the user might be confused about the `/ repo` part, since they have only supplied the owner.

Furthermore, we should only print this message if the user has supplied an owner handle on the command line. Also, if the result object has the `FromSearch` property, we should avoid printing this error message because `listResult.Owner` will always be blank there if there were no results, even if the owner name was valid.

Finally, the error message could print the user's input so that it's clear exactly what string didn't resolve to anything.

Here's what I'm proposing; please check for whether it makes sense:

```suggestion
	if opts.Owner != "" && listResult.Owner == "" && !listResult.FromSearch {
		return fmt.Errorf("the owner handle %q was not recognized as either a GitHub user or an organization", opts.Owner)
```

Reply:
I misunderstood the usage of owner flag and was wondering sometimes owner is blank. It's more clear now. Thanks for the explanation.